modules/server/src/network/index.ts

```diff
@@ -117,14 +122,7 @@ const fetchRemoteSchemas = async ({
 
 	const networkQueries = await Promise.allSettled(networkQueryPromises);
 	const configs = networkQueries
-		.filter(
-			(
-				result,
-			): result is PromiseFulfilledResult<{
-				config: NetworkAggregationConfigInput;
-				gqlResponse: GQLTypeQueryResponse;
-			}> => result.status === 'fulfilled',
-		)
+		.filter(fulfilledPromiseFilter<NetworkQueryResult>)
```

Comment:
as of aTS 5.5 you shouldn't need a function with `input is Xyz`, the type predicate will be inferred by TS. Nothing wrong with this just FYI